core-java-modules/core-java-collections-maps-7/src/test/java/com/baeldung/map/LimitMaxSizeHashMapByLinkedHashMapUnitTest.java

```diff
@@ -0,0 +1,43 @@
+package com.baeldung.map;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class LimitMaxSizeHashMapByLinkedHashMapUnitTest {
+
+    private final int MAX_SIZE = 4;
+    private LinkedHashMap<Integer, String> linkedHashMap;
+
+    @BeforeEach
```

Comment:
Since there's only one test method, we could just move all this logic into the test.

Reply:
Ok, I fixed this in a new commit